### ISSUE TITLE
Add "how can we help" or contact page for scientist from different fields

### ISSUE THREAD

**Issue by prabhant** [openml/openml.org#231]

**Comment by ihsaan-ullah** [comment]
Is this meant for specific sicentists or general contact us with title and description?

**Comment by prabhant** [comment]
It is supposed to be a contact us page but is designed for scientists from different fields who want to integrate their data in OpenML. It is supposed to be a little different from a simple contact us page where users or developers can email us.

**Comment by ihsaan-ullah** [comment]
I am not sure what we want to add on this page. I am keeping this on hold until some more requirements come about this.

**Comment by prabhant** [comment]
Yes, this issue is meant to discuss what kind of content we want to put on this page. Maybe the input of a scientist (outside ML) might be useful here
